src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Styles/styles.scss

```diff
@@ -75,6 +75,8 @@ a {
     cursor: pointer;
 
     a {
+        display: flex;
+        align-items: center;
         color: inherit;
         font-family: inherit;
 
```

Comment:
since this class (component-list-item) is only used for the HomePage components, shouldn't we move it to the `HomePage.razor.scss` file?

Reply:
As a convention, we moved all CSS of bit component classes to the `style.scss` file, regardless of whether it is for a specific page.
I think this is the reason.
What do you think? @maryamhdr

Reply:
@sajadkardel 
Could you check if the `component-list-item` class be declared in the `HomePage.razor.scss` file, the items get the correct styles or not?

Reply:
@sajadkardel 
no we didn't.
we moved some classes but the reason is not what you explained. plz test it and share the result with us.
but I think since the isolated CSS classes have issues with components, I think it won't work for this situation.

Reply:
I tested it. didn't work in the `HomePage.razor.scss` file @maryamhdr @msynk